constants.js

```diff
@@ -0,0 +1,89 @@
+    [["maxperiodsranked"], getBLCString("maxperiods", "ranked")],
+    [["maxperiodsunranked"], getBLCString("maxperiods", "unranked")],
+    [["minperiods"], getBLCString("minperiods", "")],
+    [["minperiodsranked"], getBLCString("minperiods", "ranked")],
+    [["minperiodsunranked"], getBLCString("minperiods", "unranked")],
+    [["ban"], "bans"],
+    [["banranked"], "bansranked"],
+    [["banunranked"], "bansunranked"],
+    [["boardsize"], "boardsizes"],
+    [["boardsizeranked"], "boardsizesranked"],
+    [["boardsizeunranked"], "boardsizesunranked"],
+    [["boardsizewidths", "boardsizewidthsranked", "boardsizewidthsunranked",
+        "boardsizeheights", "boardsizeheightsranked", "boardsizeheightsunranked"], "boardsizes"],
+    [["komi"], "komis"],
+    [["komiranked"], "komisranked"],
+    [["komiunranked"], "komisunranked"],
+    [["speed"], "speeds"],
+    [["speedranked"], "speedsranked"],
+    [["speedunranked"], "speedsunranked"],
+    [["timecontrol"], "timecontrols"],
+    [["timecontrolranked"], "timecontrolsranked"],
+    [["timecontrolunranked"], "timecontrolsunranked"]
+];
+
+const ogsPvAIs = ["LeelaZero", "Sai", "KataGo", "PhoenixGo", "Leela"];
+
+const rankedUnrankedOptions = [
+    { name: "bans" },
+    { name: "boardsizes", default: "9,13,19" },
+    { name: "komis", default: "automatic" },
+    { name: "speeds", default: "all" },
+    { name: "timecontrols", default: "fischer,byoyomi,simple,canadian" },
+    { name: "proonly" },
+    { name: "nopause" },
+    { name: "nopauseonweekends" },
+    { name: "noautohandicap" },
+    { name: "minrank" },
+    { name: "maxrank" },
+    { name: "minhandicap" },
+    { name: "maxhandicap" },
+    { name: "minmaintimeblitz", default: 15 }, // 15 seconds
+    { name: "minmaintimelive", default: 60 }, // 1 minutes
+    { name: "minmaintimecorr", default: 259200 }, // 3 days
+    { name: "minperiodsblitz", default: 3 },
+    { name: "minperiodslive", default: 3 },
+    { name: "minperiodscorr", default: 3 },
+    { name: "minperiodtimeblitz", default: 5 }, // 5 seconds
+    { name: "minperiodtimelive", default: 10 }, // 10 seconds
+    { name: "minperiodtimecorr", default: 14400 }, // 4 hours
+    { name: "maxmaintimeblitz", default: 300 }, // 5 minutes
+    { name: "maxmaintimelive", default: 7200 }, // 2 hours
+    { name: "maxmaintimecorr", default: 604800 }, // 7 days
+    { name: "maxperiodsblitz", default: 20 },
+    { name: "maxperiodslive", default: 20 },
+    { name: "maxperiodscorr", default: 10 },
+    { name: "maxperiodtimeblitz", default: 10 }, // 10 seconds
+    { name: "maxperiodtimelive", default: 120 }, // 2 minutes
+    { name: "maxperiodtimecorr", default: 259200 } // 3 days
+];
+
+function getBLCString(optionName, rankedUnranked) {
+    return `${optionName}blitz${rankedUnranked}, --${optionName}live${rankedUnranked} `
+           + `and/or --${optionName}corr${rankedUnranked}`;
+}
+
+exports.constants = { droppedOptions, ogsPvAIs, rankedUnrankedOptions };
```

Comment:
can we not just drop the `constants` and just do
```Js
exports = { droppedOptions, ogsPvAIs, rankedUnrankedOptions };
```
and then also remove `constants` when we import?

Reply:
ah i see, but then the exports become a bit messier no?

considering we use these only in config.js and getArgv.js, i find it clearer to wrap them all in the same module no ?

similar to the issue we had in #342 when we wanted to create a sub directory :sweat_smile: